### ISSUE TITLE
DVB-S tunning with DVB-API < 5.5 fails

### ISSUE THREAD

**Issue by lars18th** [catalinii/minisatip#142]
Hi,

This bug is related to these two troubles:
 http://github.com/catalinii/minisatip/issues/98
 http://github.com/catalinii/minisatip/issues/104

Before the bug I explain the environment: After some more test I discover that in my Debian 7 x64 the vtuner sat>ip client is working, but only with DVB-S2 signals, not with DVB-S.

So, the bug is: when a frequency with tuning mode DVB-S is requested no tuner is found, but when is requested DVB-S2 is found.

The problem is: when the tuner (with DVBAPI < 5.5) is detected is assigned a msys=6 (DVB-S2), then when searching for msys=5 (DVB-S), no tuner is returned.

Here a log of the fail:
```
[03/07 15:45:09.4174]: Play for stream 0, type 2, rsock 15, adapter -1, sock_id 8 handle 14
[03/07 15:45:09.4174]: get free adapter -1 - a[0] => e:1 m:-1 sid_cnt:0 f:0 pol=0
[03/07 15:45:09.4174]: no adapter found for f:10744000 pol:2 msys:5
[03/07 15:45:09.4174]: Dumping adapters:
[03/07 15:45:09.4174]: 0|f: 0 sid_cnt:0 master_sid:-1 del_sys: dvbs2 undefined undefined
[03/07 15:45:09.4174]: 1|f: 0 sid_cnt:0 master_sid:-1 del_sys: dvbt undefined undefined
[03/07 15:45:09.4174]: 2|f: 0 sid_cnt:0 master_sid:-1 del_sys: dvbt undefined undefined
```

And the corresponding lines of code are:
http://github.com/catalinii/minisatip/blob/a6dbec64afc43a49f8e399f3c570ad8f9b0227f3/adapter.c#L310

Please, can you fix this bug?
Thank you!

**Comment by catalinii** [comment]
Hi,

I will fix this in the next commit.

Thanks

**Comment by lars18th** [comment]
Hi Catalin,

Thank you! Waiting for the commit... ;-)

**Comment by lars18th** [comment]
Hi Catalin,

Burden fix:

adapter.c
```
int
get_free_adapter (int freq, int pol, int msys, int src)
{
        int i;
        int omsys = msys;

+        if (msys == SYS_DVBS) msys=SYS_DVBS2;
+
        i = (src > 0) ? src - 1 : 0;
```

**Comment by lars18th** [comment]
Hi Catalin,

Thank you! Fixed in last commit (v0.3.44).
